### ISSUE TITLE
Kodi smb not working

### ISSUE THREAD

**Issue by coolcars** [NSSpiral/Blackb0x#144]
Have you guys tried adding a smb (windows share folder) location under Kodi -> File Manager -> source?
I'm not able to add it properly.  It says Operation Time Out.  Does this feature work?

**Comment by jackbb72** [comment]
/+ 1

**Comment by zerochka** [comment]
don't work any internet source 

I create ftp user for kodi 
a Try smb client

**Comment by qrzeller** [comment]
No, i ended up using webdav in http so it's more cpu efficient.
